src/main/msp/msp.c

```diff
@@ -2454,11 +2470,7 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
         sbufReadU8(src); // was gyroConfigMutable()->gyro_sync_denom - removed in API 1.43
         pidConfigMutable()->pid_process_denom = sbufReadU8(src);
         motorConfigMutable()->dev.useUnsyncedPwm = sbufReadU8(src);
-#ifdef USE_DSHOT
-        motorConfigMutable()->dev.motorPwmProtocol = constrain(sbufReadU8(src), 0, PWM_TYPE_MAX - 1);
-#else
-        motorConfigMutable()->dev.motorPwmProtocol = constrain(sbufReadU8(src), 0, PWM_TYPE_BRUSHED);
-#endif
+        motorConfigMutable()->dev.motorPwmProtocol = sbufReadU8(src);
```

Comment:
Everything looks good, but can we please just add this tiny validation here?
```
        const uint8_t motorPwmProtocol = sbufReadU8(src);
        motorConfigMutable()->dev.motorPwmProtocol = (motorPwmProtocol < PWM_TYPE_MAX) ? motorPwmProtocol : PWM_TYPE_DISABLED;
```
Since we know the 10.6 configurator will corrupt the setting this tiny bit of validation will save us lots of support headaches.

Reply:
Ok, I will add the validations required for to make it possible for users of 4.2 to use 10.6.0, 10.5.0, and 10.4.1.

But this is really outside of this pull request, so can we please move on with this, and do these changes in a separate pull request?